dts/arm/nxp/nxp_imx.dtsi

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
I think the file `dts/arm/nxp/nxp_imx.dtsi` should be renamed to something more specific, for example `dts/arm/nxp/nxp_imx7d_m4.dtsi`. Unless we plan to list all cores in the file, in which case the `_m4` suffix would not be there (but here Zephyr will probably run only on M4 core, unlike in the case of `dts/arm/nxp/nxp_lpc54xxx.dtsi`, where Zephyr runs on both cores and the file lists them all).

(Now that we have multiple iMX SoC's, maybe also `dts/arm/nxp/nxp_rt.dtsi` could be renamed into something more specific, but it is not part of this pull request).

Reply:
agreed, this should be imx7d-m4.dtsi.